tools/osbuilder/tests/test_images.sh

```diff
@@ -209,7 +209,7 @@ exit_handler()
 	sudo -E kata-collect-data.sh >&2
 
 	info "processes:"
-	sudo -E ps -efwww | egrep "docker|kata" >&2
+	sudo -E ps -efwww | grep "docker|kata" >&2
```

Comment:
`egrep` stands for extended grep, so we need the `-E` to be equivalent and enable regexes:
```suggestion
	sudo -E ps -efwww | grep -E "docker|kata" >&2
```

Reply:
@sprt changes applied